src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* **Breaking change**: The same API is now exposed for `net462` and
+  `netstandard2.0` targets. `SetDbStatement` has been removed. Use
+  `SetDbStatementForText` to capture command text and stored procedure names on
+  .NET Framework.
```

Comment:
It's probably a good idea to list out the options that would only be supported on .NET and .NET core in the CHANGELOG.

Reply:
I added more text to the CHANGELOG & updated the README. Check it out, LMK what you think!

Reply:
Looks good!